datadog/import_datadog_monitor_test.go

```diff
@@ -29,16 +29,18 @@ func TestDatadogMonitor_import(t *testing.T) {
 const testAccCheckDatadogMonitorConfigImported = `
 resource "datadog_monitor" "foo" {
   name = "name for monitor foo"
-  type = "metric alert"
+  type = "query alert"
   message = "some message Notify: @hipchat-channel"
   escalation_message = "the situation has escalated @pagerduty"
 
   query = "avg(last_1h):avg:aws.ec2.cpu{environment:foo,host:foo} by {host} > 2.5"
 
   thresholds {
     ok = 1.5
-    warning = 2.3
-    critical = 2.5
+    warning           = 2.3
+    warning_recovery  = 2.2
+    critical          = 2.5
+    critical_recovery = 2.4
```

Comment:
given these fields are Optional - it'd be good to have a test without them specified (as this test was previously?)

Reply:
done